Bottom: 8b91ce8131ce460b20b6d7a62f6bd6a75bfe5b7a
Top:    aeb1a10dcbc74849dbe7130e70044db897f01ffc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:53:18 +0000

polygon1.c: Remove now unused memory allocation macro MemGet


---

diff --git a/src/polygon1.c b/src/polygon1.c
index c558843..f7db53f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -105,10 +105,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 #define DEBUG_LABEL
 #define DEBUG_ALL_LABELS
 #define DEBUG_JUMP
